cfme/services/catalogs/catalog.py

```diff
@@ -34,8 +37,14 @@ class CatalogsMultiBoxSelect(MultiBoxSelect):
 class CatalogForm(ServicesCatalogView):
     title = Text('#explorer_title_text')
 
-    name = Input(name='name')
-    description = Input(name="description")
+    name = VersionPicker({
+        "5.11": ReactTextInput(name='name'),
+        LOWEST: Input(name='name')
```

Comment:
Q: is `ReactTextInput` not working for both versions?
If working; I don't think we need this `VersionPicker`.

Reply:
It does work for both versions and I originally had it as just the `ReactTextInput` but @mshriver wanted to see it be explicit